superset-frontend/plugins/plugin-chart-echarts/src/Sunburst/controlPanel.tsx

```diff
@@ -91,6 +91,18 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'node_click',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Node Click'),
```

Comment:
Can we use a label that describes the feature it enables rather than action? Like `label: t('Enable drill down'),`?

Reply:
@kgabryje I'm wondering what's the relationship of this feature with cross-filtering and drill by? By default, charts offer the left click for cross-filtering and the right-click for the menu options. They don't offer controls to enable/disable this behavior if I'm not mistaken. I wonder if we should just add support for left/right clicks as the other chart types do.

Reply:
Oh you're right, this feature would be in conflict with cross filtering. I wonder if it would make sense to trigger drill down with some combination like ctrl + left click. Of course then we'd need some tooltip in chart itself for discoverability

Reply:
Why not implement drill down using drill by as we do for all chart types?